app/assets/javascripts/graph.js

```diff
@@ -0,0 +1,53 @@
+window.graphs = {};
+
+var Graph = (function () {
```

Comment:
I see the misuse of "graph" is quite widespread. 😆 

Never mind; I'll merge this and open a low-priority issue to adjust the names later.